crates/mockiato-codegen/src/parse/static_attr.rs

```diff
@@ -1,33 +1,32 @@
-use crate::constant::ATTR_NAME;
-use crate::spanned::SpannedUnstable;
-use crate::{Error, Result};
-use proc_macro::{Diagnostic, Level};
+use crate::constant::{ATTR_NAME, STATIC_REFERENCES_ATTR_PARAM_NAME};
+use crate::diagnostic::DiagnosticBuilder;
+use crate::result::Result;
+use syn::spanned::Spanned;
 use syn::Meta;
 
 #[cfg_attr(feature = "debug-impls", derive(Debug))]
 pub(crate) struct StaticAttr;
 
 impl StaticAttr {
     pub(crate) fn parse(meta_item: Meta) -> Result<Self> {
-        let meta_item_span = meta_item.span_unstable();
+        let meta_item_span = meta_item.span();
 
         if let Meta::Word(_ident) = meta_item {
             return Ok(Self);
         }
 
-        Err(Error::Diagnostic(
-            Diagnostic::spanned(
-                meta_item_span,
-                Level::Error,
-                format!(
-                    "#[{}(static_references) does not take any parameters",
-                    ATTR_NAME
-                ),
-            )
-            .help(format!(
-                "Example usage: #[{}(static_references)]",
-                ATTR_NAME
-            )),
-        ))
+        let error_message = format!(
+            "#[{}({}) does not take any parameters",
+            ATTR_NAME, STATIC_REFERENCES_ATTR_PARAM_NAME
+        );
+        let help_message = format!(
+            "Example usage: #[{}({})]",
```

Comment:
I'd rephrase this as "Help:". It's not really an example, but a guide on how to fix the issue.

Reply:
It already gets prefixed with “Help” afaik. How about “Correct usage:”?

Reply:
Sounds good.

Reply:
Addressed in 836805f